packages/flutter_test/lib/src/test_compat.dart

```diff
@@ -21,6 +21,9 @@ import 'package:test_api/src/backend/state.dart'; // ignore: implementation_impo
 // ignore: deprecated_member_use
 import 'package:test_api/test_api.dart';
 
+// ignore: deprecated_member_use
+export 'package:test_api/fake.dart';
```

Comment:
Should we add `show Fake` here to make it super-explicit what we are exporting?

Reply:
Done